src/Microsoft.VisualStudio.Editor.Razor/DefaultTemplateEngineFactoryService.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.VisualStudio.Editor.Razor
     internal class DefaultTemplateEngineFactoryService : RazorTemplateEngineFactoryService
     {
         private readonly static MvcExtensibilityConfiguration DefaultConfiguration = new MvcExtensibilityConfiguration(
+            RazorLanguageVersion.Version2_0,
```

Comment:
Going to do another pass in a different PR on all of the versions in this class and in `DefaultProjectExtensibilityConfigurationFactory` to update them to 2.1 (latest) since 15.7 aligns with 2.1